### PR TITLE
Fix commit --changes env=X=Y

### PR DESCRIPTION
**Pull request by jwhonce**
Signed-off-by: Jhon Honce <jhonce@redhat.com>

### CODE REVIEW

**Comment by baude** [comment]
lgtm

File: cmd/podman/shared/create.go [hunk 1]
```diff
@@ -77,7 +77,11 @@ func CreateContainer(ctx context.Context, c *GenericCLIResults, runtime *libpod.
 			writer = os.Stderr
 		}
 
-		newImage, err := runtime.ImageRuntime().New(ctx, c.InputArgs[0], rtc.SignaturePolicyPath, GetAuthFile(""), writer, nil, image.SigningOptions{}, false, nil)
+		name := ""
+		if len(c.InputArgs) != 0 {
```
Comment (mheon) [inline cmd/podman/shared/create.go]: It seems like it ought to be an error if this isn't true - we can give a much clearer error message than what pulling an empty-string image would give.

**Comment by TomSweeneyRedHat** [comment]
LGTM other then @mheon's note.  I'd be fine with a merge of this and a follow up for the error.

**Comment by rhatdan** [comment]
/lgtm
Lets merge since @jwhonce is on PTO, and someone else can add an error.

**Pull request merged**